iree/compiler/Conversion/LinalgToSPIRV/LinalgTileAndFusePass.cpp

```diff
@@ -87,7 +82,9 @@ class TileSizeCalculator {
 
   /// Set workgroup size to use.
   void setWorkgroupSize(ArrayRef<int64_t> sizes) {
-    workgroupSize.assign(sizes.begin(), sizes.end());
+    workgroupSize = {1, 1, 1};
+    for (unsigned i = 0, e = std::min<unsigned>(2, sizes.size()); i != e; ++i)
```

Comment:
can we use `e = std::min(2u, sizes.size())`

and maybe replace `i != e` with `i < e`? I think we typically use `i != e` when it's an iterator.